### ISSUE TITLE
Unable to install Reloaded after uninstalling it

### ISSUE THREAD

**Issue by Zapcat56** [Reloaded-Project/Reloaded-II#158]
I first installed reloaded a while ago to mod P4G but I haven't touched it in a long time and as I'm now modding P5R I started to have a lot of weird issues with it, I decided to uninstall and reinstall it but now I can't get it to install at all and I'm not sure if I even uninstalled it correctly.  This is the error I'm getting whenever I run the setup.exe.
![image](https://user-images.githubusercontent.com/117522676/200121980-72d720c8-b56f-470a-afc3-17e7ffa00418.png)

**Comment by Sewer56** [comment]
GitHub limits requests to their API to 60 an hour per IP address (it's a very, very low limit that causes a lot of problems to devs like us).  
One way or another, rate limit was exceeded where you are, so installer can't check [the latest Reloaded-II version](https://github.com/Reloaded-Project/Reloaded-II/releases/latest).

You can wait a short while and try again until it lets you or download `Release.zip` from the page above, and any runtimes it might ask you to once you try to launch `Reloaded-II.exe`

**Comment by Zapcat56** [comment]
Used the zip to install it and it was working fine until I tried to launch the game, now it's doing this thing where it flashes a window very rapidly that says downloading dependencies.  I closed it and relaunched and it prompted me to update Persona 5 Royal Essentials in which upon clicking download it does the exact same thing.  Can't get a screenshot due to how fast it's going.

**Comment by Sewer56** [comment]
This is probably one of those rare edgecases where ISP (or some other reason) blocks connection connection to my NuGet server, and because you're rate limited by GitHub, it can't download from there either; so it fails to download from both main server and backup.

If it doesn't resolve itself in a short while, let me know.

**Comment by Zapcat56** [comment]
That's really odd cause last time I used reloaded I had absolutely 0 issues and I was on the exact same ISP.  Maybe they changed something since then though.  I'll try it periodically for the rest of today and see if it ever manages to work.

**Comment by Sewer56** [comment]
Note: By short while I mean less than 60 minutes, i.e. before rate limit resets.

If you're still outta luck, try pasting these in your browser.
[If you're still missing any dependencies after those, let me know]

`r2:https://github.com/Sewer56/Reloaded.SharedLib.Hooks.ReloadedII/releases/download/1.14.2/Reloaded.Hooks.ReloadedII1.14.2.7z`

`r2:https://github.com/Reloaded-Project/Reloaded.Memory.SigScan/releases/download/3.1.10/Reloaded.Memory.SigScan.ReloadedII1.2.9.7z`

**Comment by Zapcat56** [comment]
Oh gotcha.  Will do then, thanks.

**Comment by Zapcat56** [comment]
Alright well I tried pasting the first one into my browser and after it did quick download I got this error message and then reloaded closed.
![FirstError](https://user-images.githubusercontent.com/117522676/200125590-52eaa9bb-4490-489d-9a36-455432c72686.PNG)
I tried the second link and it seemed to go through fine as the missing dependencies alert on launch is now just reloaded.sharedlib.hooks

I decided to try the first one again just in case and sure enough it ended up working, from what I can tell the game launches without any issues and I'm not getting any dependencies alerts anymore.  Thanks for the quick help!
